pkg/aws/core_test.go

```diff
@@ -39,6 +39,11 @@ const (
 	cloudProviderReturnedError            = "machine codes error: code = [Internal] message = [Cloud provider returned error]"
 )
 
+var (
+	affinity_const  = ec2.AffinityDefault
+	tenancy_const   = ec2.TenancyDefault
+	availZone_const = "eu-west-1a"
```

Comment:
```suggestion
	affinityConst  = ec2.AffinityDefault
	tenancyConst   = ec2.TenancyDefault
	availZoneConst = "eu-west-1a"
```

GoLang generally prefers the approach of mixedCases. Refer - https://golang.org/doc/effective_go#mixed-caps. Also, why not use `ec2.AffinityDefault` and `ec2.TenancyDefault` directly? Also please adopt the rest of the code with the same.

Reply:
I used the constants so that we could change ec2.AffinityDefault to lets say ec2.AffinityHost at just one placeand in the expect block of one of the test cases , since we have to initialzie &value, using of var affinityConst and tenancyConst seemed the only option to me, unless there's a way to create variable on the fly and assign their address to a field.